bookwyrm/tests/models/test_bookwyrm_import_job.py

```diff
@@ -0,0 +1,567 @@
+""" testing models """
+
+import json
+import pathlib
+from unittest.mock import patch
+
+from django.db.models import Q
+from django.utils.dateparse import parse_datetime
+from django.test import TestCase
+
+from bookwyrm import models
+from bookwyrm.utils.tar import BookwyrmTarFile
+from bookwyrm.models import bookwyrm_import_job
+
+
+class BookwyrmImport(TestCase):
+    """testing user import functions"""
+
+    def setUp(self):
+        """setting stuff up"""
+        with patch("bookwyrm.suggested_users.rerank_suggestions_task.delay"), patch(
+            "bookwyrm.activitystreams.populate_stream_task.delay"
+        ), patch("bookwyrm.lists_stream.populate_lists_task.delay"), patch(
+            "bookwyrm.suggested_users.rerank_user_task.delay"
+        ):
+
+            self.local_user = models.User.objects.create_user(
+                "mouse",
+                "mouse@mouse.mouse",
+                "password",
+                local=True,
+                localname="mouse",
+                name="Mouse",
+                summary="I'm a real bookmouse",
+                manually_approves_followers=False,
+                hide_follows=False,
+                show_goal=True,
+                show_suggested_users=True,
+                discoverable=True,
+                preferred_timezone="America/Los Angeles",
+                default_post_privacy="public",
+            )
+
+            self.rat_user = models.User.objects.create_user(
+                "rat", "rat@rat.rat", "password", local=True, localname="rat"
+            )
+
+            self.badger_user = models.User.objects.create_user(
+                "badger",
+                "badger@badger.badger",
+                "password",
+                local=True,
+                localname="badger",
+            )
+
+            self.work = models.Work.objects.create(title="Test Book")
+
+            self.book = models.Edition.objects.create(
+                title="Test Book",
+                remote_id="https://example.com/book/1234",
+                openlibrary_key="OL28216445M",
+                parent_work=self.work,
+            )
+
+        archive_file = pathlib.Path(__file__).parent.joinpath(
+            "../data/bookwyrm_account_export.tar.gz"
+        )
+        self.tarfile = BookwyrmTarFile.open(
+            mode="r:gz", fileobj=open(archive_file, "rb")
+        )
```

Comment:
> there are some pylint errors in test_bookwyrm_import_job which indicate I probably need to refactor a test there but I can't figure out what I should be doing

Not sure if this was about this warning, but here you could do?:

```suggestion
        with open(archive_file, "rb") as fileobj:
            self.tarfile = BookwyrmTarFile.open(mode="r:gz", fileobj=fileobj)
```